frappe/email/doctype/email_account/email_account.py

```diff
@@ -123,7 +123,10 @@ def get_incoming_server(self, in_receive=False):
 		try:
 			email_server.connect()
 		except (error_proto, imaplib.IMAP4.error), e:
-			if in_receive and ("authentication failed" in e.message.lower() or "log in via your web browser" in e.message.lower()):
+			message = e.message.lower().replace(" ","")
+			if in_receive and ("authenticationfail" in message #abbreviated to work with both failure and failed
+			or "loginviayourwebbrowser" in message or "loginfailed" in message
+			or "-err[auth]" in message or "-errtemporaryerror" in message): #temporary error to deal with godaddy
```

Comment:
better way
`any(map(lambda t: t in message, ['authenticationfail', 'loginviayourwebbrowser', 'loginfailed', 'err[auth]', 'errtemporaryerror']))`